### PR TITLE
Server for integration tests

### PR DESCRIPTION
**Pull request by FinalAngel**
**just testing currently against travis**

### CODE REVIEW

**Comment by intellectronica** [comment]
:+1:

File: aldryn_events/tests/frontend/integration/specs/spec.events.crud.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+ * @copyright: http://www.divio.ch
+ */
+
+'use strict';
+/* global describe, it, browser */
+
+// #############################################################################
+// INTEGRATION TEST
+describe('Aldryn Events tests: ', function () {
+    var eventsPage = require('../pages/page.events.crud.js');
```
Comment (vxsx) [inline aldryn_events/tests/frontend/integration/specs/spec.events.crud.js]: can we move it out of describe?
requires in general should be on top of the file
we are going to go into the ecmascript 6 land and and reality is that possibility of this behaviour would go away soon so it's better to be consistent

Comment (FinalAngel) [inline aldryn_events/tests/frontend/integration/specs/spec.events.crud.js]: @Marketionist can you do that also on your next PR?

Comment (Marketionist) [inline aldryn_events/tests/frontend/integration/specs/spec.events.crud.js]: Yes, will do

File: aldryn_events/tests/frontend/integration/pages/page.events.crud.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+ */
+
+'use strict';
+/* global by, element, browser, expect */
+
+// #############################################################################
+// INTEGRATION TEST PAGE OBJECT
+
+var eventsPage = {
+    site: 'http://127.0.0.1:8000/en/',
```
Comment (FinalAngel) [inline aldryn_events/tests/frontend/integration/pages/page.events.crud.js]: is this required here @vxsx @Marketionist as the base url is added here: https://github.com/aldryn/aldryn-events/pull/109/files#diff-b9e12334e9eafd8341a6107dd98510c9R104

**Comment by vxsx** [comment]
LGTM :tada:

File: aldryn_events/tests/frontend/integration/pages/page.events.crud.js (hunk 2 above)
Comment (vxsx) [inline aldryn_events/tests/frontend/integration/pages/page.events.crud.js]: actually no, this should be `/`

File: aldryn_events/tests/frontend/protractor.conf.js [hunk 3]
```diff
@@ -20,22 +20,29 @@ var config = {
...
     jasmineNodeOpts: {
         showColors: true,
-        defaultTimeoutInterval: 30000
+        defaultTimeoutInterval: 240000
     }
+
 };
 
-if (process.env.SAUCE_USERNAME && process.env.SAUCE_ACCESS_KEY) {
+if (process.env.CI || process.env.SAUCE_USERNAME && process.env.SAUCE_ACCESS_KEY) {
```
Comment (vxsx) [inline aldryn_events/tests/frontend/protractor.conf.js]: uhm, actually this should be reverted, forgot this :(

**Comment by intellectronica** [comment]
:+1:

**Pull request merged**